composer.json

```diff
@@ -36,11 +37,15 @@
         "php-http/mock-client": "^0.3.2",
         "phpunit/phpunit": "^5.7 || ^6.0",
         "symfony/process": "^2.3 || ^3.0 || ^4.0",
-        "symfony/http-kernel": "^2.3 || ^3.0 || ^4.0"
+        "symfony/http-kernel": "^2.3 || ^3.0 || ^4.0",
+        "symfony/psr-http-message-bridge": "^1.0",
+        "zendframework/zend-diactoros": "^1.7"
     },
     "suggest": {
         "friendsofsymfony/http-cache-bundle": "For integration with the Symfony framework",
-        "monolog/monolog": "For logging issues while invalidating"
+        "monolog/monolog": "For logging issues while invalidating",
+        "symfony/psr-http-message-bridge": "If you need direct routing for Symfony.",
```

Comment:
we should be more specific here: "For simplified setup for cache invalidation in a single node system with symfony HttpCache"

Reply:
You're right, done in b04699281a4eda1121edbd41a8331f7afdbf116e.